### PR TITLE
Apply the toolchain version update

### PR DESCRIPTION
**Pull request by HONGCHAEHEE**
- GCC version has been updated from 4.9.3 to 6.3.1.
- refer to https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads/6-2017-q1-update for download and information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ Please find project details at [Wiki](https://github.com/Samsung/TizenRT/wiki) e
 ## Quick Start
 ### Getting the toolchain
 
-Get the build in binaries and libraries, [gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2](https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q3-update)  
-Untar the gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2 and export the path like
+Get the build in binaries and libraries, [gcc-arm-none-eabi-6-2017-q1-update-linux.tar.bz2](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads)
```
Comment (jeongchanKim) [inline README.md]: I'm worried that it open not proper TizenRT supported version but the latest version.

Comment (HONGCHAEHEE) [reply README.md]: The latest version and the other versions are in the same link.

**Comment by sunghan-chang** [comment]
@pradeepns85 Does it mean toolchain changes? ```major changes happened from previous version to latest version.```

**Comment by pradeepns85** [comment]
@sunghan-chang, Ya I meant "official Link " to see the changes. If possible we can mention Few major benefits of using new version for TizenRT.  
Thanks,
Pradeep

**Comment by sunghan-chang** [comment]
@HONGCHAEHEE Could you remove below tab?
https://github.com/Samsung/TizenRT/blob/master/docs/TroubleShooting.md#issues-on-gnu-toolchain
After merging this PR, above issue will not happen

**Comment by seinfra** [comment]
Target : [c0db26855e30a5eed268b1fd374d8010980175b7] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c0db26855e30a5eed268b1fd374d8010980175b7] - Code Rule Check (C++) OK.

**Pull request merged**